tools/esdlcmd/esdlcmd_monitor.cpp

```diff
@@ -973,14 +973,27 @@ class EsdlMonitorCmd : public EsdlConvertCmd
 
         xform->setParameter("diffmode", "'Monitor'");
         xform->setParameter("diffaction", "'Create'");
+
+        xform->setParameter("platform", "'roxie'");
```

Comment:
Same comment about setting a standard for $platform values (all upper, all lower, camel, etc)